frontends/pytorch/test/lit.cfg.py

```diff
@@ -58,10 +58,11 @@
 config.npcomp_tools_dir = os.path.join(config.npcomp_obj_root, 'bin')
 
 # Tweak the PATH to include the tools dir.
+npcomp_python_dir = "python" if config.npcomp_built_standalone else "tools/npcomp/python"
```

Comment:
I found that the `npcomp` python artifacts were placed under `tools/npcomp/python` when doing an external build. A couple things about this: 1) I'm not sure if there is a better way to use CMake to put the python just in `build/python` when doing an external build, 2) I think the `npcomp` in the `tools/npcomp/python` path comes from the name passed to the `LLVM_EXTERNAL_PROJECTS` flag, so this may not be guaranteed to match for everyone's setup. I have a feeling there is a better way to achieve this line of python with CMake, but I haven't figured it out yet.

Reply:
Let's land it in a working state how you have it. Then I/someone can try to find a more targeted place to put it. I'd hope we can just emit all of the python artifacts into the top level `build/python` directory: even across projects, they form a namespace that must not overlap, so having everything write to that directory is ok (CMake will complain if two projects then try to write the same thing into the python namespace).